css-display-3/display-contents-tr-001-ref.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Reftest Reference</title>
+<link rel="author" title="Rune Lillesveen" href="mailto:rune@opera.com">
+<p>You should see the word PASS below.</p>
+<table cellpadding="0" cellspacing="0">
+    <td>P<span>A</span></td>
+    <td>S<span>S</span></td>
+</table>
```

Comment:
Fair enough, amended to include it. And yeah, I think it was a shaping bug because the two first letters were shaped correctly, and the difference between the other two letters was _really_ minimal too.